lib/routes/user/pay_nearby/pay_nearby_page.dart

```diff
@@ -58,6 +59,7 @@ class PayNearbyState extends State<_PayNearbyPage> {
                     if (_formKey.currentState.validate()) {
                       _formKey.currentState.save();
                       widget._invoiceBloc.payBlankAmount = _amountToSendSatoshi;
+                      widget._invoiceBloc.description = _descriptionController.text;
```

Comment:
> What is the purpose of this "description" field? I also don't see it assigned anywhere.

It is assigned here.

Reply:
It is wrong to assign a bloc variable from outside. Everything in the bloc should be private except for the stream interfaces.
I see also previously we had payBlankAmount behaves like this. Let's fix it.